src/DBFTPlugin/DBFTPlugin.csproj

```diff
@@ -5,15 +5,37 @@
     <RootNamespace>Neo.Consensus</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <CustomCommands>
+      <CustomCommands>
+        <Command>
+          <type>BeforeBuild</type>
+          <command>dotnet-format</command>
+          <externalConsole>True</externalConsole>
+        </Command>
+      </CustomCommands>
+    </CustomCommands>
+  </PropertyGroup>
```

Comment:
If we add this we should add it to all the projects, I think that it can be removed

Reply:
lol, i just think this would save a lot of work in format updating ( or is it just me who make a lot of format errors?).  it wont hurt even if we add it to all projects, right?

Reply:
I think that it's not a bad change, but seems weird if only in one project was added, isn't it?

Reply:
> @shargon lol, i just think this would save a lot of work in format updating ( or is it just me who make a lot of format errors?). it wont hurt even if we add it to all projects, right?

I think you could add to the whole project.